privacyidea/api/before_after.py

```diff
@@ -332,9 +332,9 @@ def privacyidea_error(error):
     These are not that critical exceptions.
     """
     if "audit_object" in g:
-        g.audit_object.log({"info": unicode(error)})
+        g.audit_object.log({"info": str(error)})
```

Comment:
Hm sorry, I missed this earlier: Might this cause problems under Python 2, because we now pass bytestrings where we used to pass unicodes? I *think* we should be fine, but I'm not really sure :-)

Reply:
You are right, i tried to pass the `error.message` here but it is not the same as `str(error)`. This gets passed to the Audit-Model, which uses `db.String` for the column type instead of `db.Unicode`.
So i will change here to `six.text_type(error)` and we should change the column types in the Audit table.

Reply:
fixed in dbfada63

Reply:
Thanks!